chain/network/src/types.rs

```diff
@@ -334,7 +221,8 @@ pub enum PeerMessage {
     /// Gracefully disconnect from other peer.
     Disconnect,
     Challenge(Challenge),
-    HandshakeV2(HandshakeV2),
+    /// Deprecated
+    Unused,
```

Comment:
maybe let's call it Unused_HandshakeV2 (so that we remember what used to be here)

(or with _ prefix)